samples/SocketsSample/project.json

```diff
@@ -55,7 +55,7 @@
   },
 
   "scripts": {
-    "precompile": [ "tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
+    "precompile": [ "../../bin/nodejs/node_modules/.bin/tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
```

Comment:
It would break if you haven't run ./build.sh before. I tried to get a local TypeScript to work cleanly but I was having trouble with `PATH`.

Right now we need something to unblock the CI. Eventually I think we'll want the TypeScript code to actually be in a TypeScript Project so we can reference it from other places.